devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -163,6 +164,21 @@ def configureHealthCheck() {
 def configure() {
 	log.debug "configure()"
 	configureHealthCheck()
-	//the orvibo switch will send out device anounce message at ervery 2 mins as heart beat,setting 0x0099 to 1 will disable it.
-	return zigbee.writeAttribute(0x0000, 0x0099, 0x20, 0x01, [mfgCode: 0x0000])
+
+	if (isOrvibo()) {
+		//the orvibo switch will send out device anounce message at ervery 2 mins as heart beat,setting 0x0099 to 1 will disable it.
+		return zigbee.writeAttribute(0x0000, 0x0099, 0x20, 0x01, [mfgCode: 0x0000])
+	} else {
+		// turn off switches during paring for Aurora
+		def cmd = []
+		cmd << childOff("${device.deviceNetworkId}:02}")
+		cmd << off()
+
+		cmd.each { sendHubCommand(it) }
+	}
 }
+
+private Boolean isOrvibo() {
+	device.getDataValue("model") == "074b3ffba5a045b7afd94c47079dd553"
```

Comment:
might be more legible to change this to `isAurora()`

Reply:
I would leave isOrvibo() - since it's a bit unusual device behaviour -> 
`//the orvibo switch will send out device anounce message at ervery 2 mins as heart beat,setting 0x0099 to 1 will disable it.
return zigbee.writeAttribute(0x0000, 0x0099, 0x20, 0x01, [mfgCode: 0x0000])`

in `else` statement would be: `return response(refresh())` for this Aurora (and probably other devices that would be supported by this DTH in the future) .
@greens What do You think about it Steven ?

Reply:
sure